src/player.js

```diff
@@ -0,0 +1,40 @@
+function Player(name) { 
+  this.name = name;
+  this._highScores = []; 
+}
+
+Player.prototype = {
+  isHighscore() {
+    let isHighscore = false;
+    if (this._highScores.length === 0) {
+        isHighscore = true;
+    } else {
+        //get the score last in the list
+        const lastScore = this._highScores[this._highScores.length - 1].score;
+        //check if highscore
+        if (parseFloat(this._highScores.score) < parseFloat(lastScore) || this._highScores.length < 2) {
```

Comment:
`this._highScores` is an array, so what is `this._highScores.score` referring to?

Reply:
Also, you are checking the length of the high score array is less than 2 here, and you have already checked if it is zero above - if you check that the length is less than 2 in the first statement, then you won't need to check it a second time here

Reply:
Your first comment...because I'm pushing an object in to the array with quiz & score keys, I thought this._highScores.score accessed the score property? :confused: